docs/installation.md

```diff
@@ -3,7 +3,7 @@
 Run below command to install latest version of aeternity node.
 
 ```bash
-curl -O https://raw.githubusercontent.com/aeternity/aeternity/v2.0.0/scripts/install.sh && chmod +x install.sh && ./install.sh 2.0.0
+curl -O https://raw.githubusercontent.com/aeternity/installer/master/scripts/install.sh && chmod +x install.sh && ./install.sh 2.0.0
```

Comment:
let's keep using tags, once https://github.com/aeternity/installer/pull/2/ is merged we should tag `v1.0.0`